tests/components/electric_kiwi/test_config_flow.py

```diff
@@ -122,6 +109,11 @@ async def test_existing_entry(
         DOMAIN, context={"source": config_entries.SOURCE_USER, "entry_id": DOMAIN}
     )
 
+    aioclient_mock.get(
+        "https://api.electrickiwi.co.nz/session/",
+        json=load_json_value_fixture("session.json", DOMAIN),
+    )
```

Comment:
I am not sure about the state of the rest of the tests, but ideally we don't mock direct aiohttp calls, but rather patch out the library.

(Although, with oauth there's a part where HA makes calls to an endpoint, in those cases its fine)